obspy/core/tests/test_stream.py

```diff
@@ -1948,6 +1948,16 @@ def test_read_url_via_network(self):
         tr = read('https://examples.obspy.org/test.sac', headonly=True)[0]
         self.assertEqual(tr.data.size, 0)
 
+    def test_read_path(self):
+        """
+        Test for reading a pathlib object.
+        """
+        base_path = Path(__file__).parent / 'data'
+        data_path = base_path / 'IU_ULN_00_LH1_2015-07-18T02.mseed'
+        assert data_path.exists()
+        st = read(data_path)
+        assert isinstance(st, Stream)
```

Comment:
historically we used to go with unittest's `self.assertTrue()` but personally I don't mind the `assert`s, they're shorter and less convoluted and I don't think they should have any significant downsides as opposed to the unittest style

Reply:
@megies, your right. I have just become so accustomed to pytest I forgot to do that here, I will fix it.